aioesphomeapi/api.proto

```diff
@@ -704,6 +705,16 @@ enum ClimateAction {
   CLIMATE_ACTION_DRYING = 5;
   CLIMATE_ACTION_FAN = 6;
 }
+enum ClimatePreset {
+  CLIMATE_PRESET_NONE = 0;
```

Comment:
Shoudn't this be removed, and HOME is to be set to 0?

Reply:
Correct, but this PR also implements https://github.com/esphome/esphome/pull/1951 (adds `NONE` preset, because HA has it too and NONE would be a good 0 value for an enum; for example if you have a device with say only eco and boost presets, but not home/away)

Reply:
Huh, you're right - weird that it's missing there.

Anyway, adding it can't hurt - and now's the best time to add it without a breaking change

Reply:
Indeed, I can see that this was introduced in this commit: https://github.com/home-assistant/core/commit/50d4921d0a2ddee4503e9e260cd65a767902cadd#diff-bc3dca6e297a377d1bb686440fe7891a93ec7f87c3a5fcaa8dca8aa3bf157fac, but they most likely forgot to update the docs.

Reply:
Also, I see that in https://github.com/esphome/esphome/pull/1941 HOME was still the default with 0, so that may need to be updated as well.